test/integration/addons_test.go

```diff
@@ -708,3 +710,53 @@ func validateGCPAuthAddon(ctx context.Context, t *testing.T, profile string) {
 		}
 	}
 }
+
+// validateInAccelAddon tests the inaccel addon by trying a vadd
```

Comment:
This is removed in https://github.com/kubernetes/minikube/commit/1f9d28bd4477caabc45d33a4bea17a307cc2b219. Can you please fix it?